cmd/commento/main.go

```diff
@@ -17,6 +17,10 @@ func main() {
 		Die(err)
 	}
 	
+	if err := godotenv.Load(); err != nil {
```

Comment:
Fantastic, then go ahead and remove the aforementioned block to avoid overwriting the port with a hardcoded value.